lib/views/frontend/solidus_paypal_commerce_platform/payments/_payment.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if payment.source.respond_to?(:paypal_email) %>
+  <br />
+  <%= payment.source.paypal_email %>
```

Comment:
Should we add some kind of label before this?

Reply:
Currently it looks like this: 
![image](https://user-images.githubusercontent.com/5720486/86827115-390c0c00-c057-11ea-878a-94e1278399d1.png)

with `Payment Email` (since `PayPal Email` might be confusing if they didn't log in to PayPal):
![image](https://user-images.githubusercontent.com/5720486/86827515-b46dbd80-c057-11ea-833c-2144307ecf1f.png)

Just `Email` could work too, could be a bit confusing if they think we're referring to the order email, but it's also under the `payment` section so maybe that's self-explanatory. What do you think?

Reply:
"Payment Email" looks great! 👍